TimberAPI/EntityLinkerSystem/EntityActions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using TimberbornAPI.EntityActionSystem;
+using UnityEngine;
+
+namespace TimberbornAPI.EntityLinkerSystem
+{
+    public class EntityActions : IEntityAction
+    {
+        /// <summary>
+        /// Apply the Linker to every object
+        /// </summary>
+        /// <param name="entity"></param>
+        public void ApplyToEntity(GameObject entity)
+        {
+            entity.AddComponent<EntityLinker>();
```

Comment:
Should we just do Building maybe?

Reply:
Hmm, not sure what other GameObjects would be good to use this with but I'm all ears

Reply:
Are those waterpoles, platforms(maybe not yet) also buildings?

Reply:
Limiting it just to Buildings makes sense. @KYPremco Buildings do include stream gauges, platforms, even paths and natural slopes